### ISSUE TITLE
Keyword length exceeds storage limit

### ISSUE THREAD

**Issue by douya520** [hievalt/ElementHider#5]
I encountered a problem. It shows ‘Maximum character amount exceeded! Keywords cannot be saved while character limit is not met. (7614 / 7602)’

**Comment by hackeranmol** [comment]
Exactly. I thought i found what i need but it does not work when the keywords are too high 

Maximum character amount exceeded! Keywords cannot be saved while character limit is not met. (36700 / 19)

Please fix this @hievalt
